docs/herramientas-para-desarrolladores.md

```diff
@@ -4,66 +4,252 @@ title: Nuestras Herramientas Favoritas
+
+[Apollo GitHub](https://github.com/apollographql)
+
+* * *
+
+## Axios
+
+### ¿Qué es Axios?
+
+Axios es una biblioteca de JavaScript muy popular que se puede usar para realizar solicitudes HTTP, funciona tanto en las plataformas Browser como en Node.js. Es compatible con todos los navegadores modernos, incluido el soporte para IE8 y superior.
+
+### ¿Por qué usamos Axios?
+
+1. Soporta navegadores más antiguos.
+2. Tiene una forma de cancelar solicitudes.
+3. Tiene una forma de establecer un tiempo de espera de respuesta.
+4. Tiene protección CSRF incorporada.
+5. Admite el progreso de la carga.
+6. Realiza una transformación automática de datos JSON.
+7. Funciona en Node.js
+
+### Instalación de Axios
+
+[Instalación de Axios](https://styde.net/solicitudes-http-con-axios/)
+
+[Axios GitHub](https://github.com/axios/axios)
+
+* * *
+
 ## EOSIO
 
-EOSIO es una cadena de bloques diseñada para facilitar el funcionamiento de aplicaciones descentralizadas (dapp).
+### ¿Qué es EOSIO?
+
+EOSIO es un software de código abierto que permite el lanzamiento de redes de blockchain altamente configurables en las que los desarrolladores y empresarios pueden ejecutar aplicaciones de blockchain de alto rendimiento. Para obtener más información sobre EOSIO, visite [aprender EOS](https://guias.eoscostarica.io/docs/aprender-eosio/protocolo-eosio).
+
+### ¿Por qué usamos EOSIO?
+
+Aquí hay algunas razones por las que usamos EOSIO:
+
+1. **Es escalable**. Para las empresas que necesitan realizar transacciones con miles de clientes simultáneamente, por ejemplo, esta velocidad de transacción es insuficiente. EOSIO, por otro lado, ha sido evaluado para procesar más de 4.000 transacciones por segundo en su blockchain pública, lo que significa que es 200 veces más rápido que su competidor más cercano y eso es solo la red pública.
+
+2. **Es rápido**. Al usar aplicaciones EOSIO, los consumidores y las empresas ni siquiera necesitan saber que están usando una "aplicación blockchain"; todo lo que saben es que cualquier dato que hayan ingresado para cualquier transacción es más seguro pero no más lento que su aplicación normal que no es blockchain.
+
+3. **Es prácticamente gratis**. A diferencia de otros protocolos de blockchain, EOSIO ofrece un modelo de costos más favorable para consumidores y desarrolladores, ya que elimina la necesidad de tarifas de transacción.
+
+4. **Es verde**. Una de las quejas más comunes que escuchas sobre la tecnología blockchain es lo cara y nociva para el medio ambiente que es. De hecho, muchas plataformas blockchain requieren una cantidad sustancial de electricidad para ejecutar las computadoras necesarias para administrar la base de datos distribuida.
+
+>[Más información](https://block.one/news/four-reasons-why-developers-and-enterprises-are-looking-at-the-eosio-blockchain-protocol/#:~:text=Unlike%20other%20blockchain%20protocols%2C%20EOSIO,apps%20are%20free%20to%20use.)
```

Comment:
Not sure about use **>** to highlight the link, that is for quotenotes

Reply:
This is how it was in the English version, so I left the same structure.

But which one do you recommend?